mysql/datadog_checks/mysql/data/conf.yaml.example

```diff
@@ -21,230 +29,280 @@ instances:
-    ## `cert`: path to your cert file.
-    ## `ca`: path to your ca file.
-    #
-    # ssl:
-    #   key: <KEY_FILE_PATH>
-    #   cert: <CERT_FILE_PATH>
-    #   ca: <CA_PATH_FILE>
+        ## @param key - string - optional
+        ## Path to a key file.
+        #
+        # key: <KEY>
+
+        ## @param cert - string - optional
+        ## Path to a cert file.
+        #
+        # cert: <CERT>
+
+        ## @param ca - string - optional
+        ## Path to a CA bundle file.
+        #
+        # ca: <CA>
 
     ## @param use_global_custom_queries - string - optional - default: true
     ## How `global_custom_queries` should be used for this instance. There are 3 options:
     ##
-    ## 1. true - `global_custom_queries` override `custom_queries`.
-    ## 2. false - `custom_queries` override `global_custom_queries`.
-    ## 2. extend - `global_custom_queries` are used in addition to any `custom_queries`.
+    ## * 'true' - `global_custom_queries` override `custom_queries`.
+    ## * 'false' - `custom_queries` override `global_custom_queries`.
+    ## * 'extend' - `global_custom_queries` are used in addition to any `custom_queries`.
     #
-    # use_global_custom_queries: true
+    # use_global_custom_queries: 'true'
 
-    ## @param custom_queries - list - optional
-    ## Each query must have 2 fields:
-    ##
-    ## 1. query - The SQL to execute. It can be a simple statement or a multi-line script.
-    ## 2. columns - The list representing each column, ordered sequentially from left to right.
-    ##              The number of columns must equal the number of columns returned in the query.
-    ##              There are 2 required pieces of data:
-    ##                a. name - The suffix to append to `clickhouse.` to form
-    ##                          the full metric name. If `type` is `tag`, this column is
-    ##                          considered a tag and applied to every
-    ##                          metric collected by this particular query.
-    ##                b. type - The submission method (gauge, monotonic_count, etc.).
-    ##                          Use `tag` to tag each metric in the row
-    ##                          with the name and value of the item in this column.
-    ##                          Use the `count` type to perform aggregation for queries that
-    ##                          return multiple rows with the same or no tags.
-    ## 3. tags (optional) - A list of tags to apply to each metric.
-    #
-    # custom_queries:
-    #   - query: |  # Use the pipe if you require a multi-line script.
-    #       SELECT foo,
-    #              COUNT(*)
-    #       FROM table.events
-    #       GROUP BY foo
-    #     columns:
-    #       # Columns without a name are ignored. Add additional columns to skip:
-    #       # - {}
-    #       - name: foo
-    #         type: tag
-    #       - name: event.total
-    #         type: gauge
-    #     tags:
-    #       - test:mysql
```

Comment:
IMHO we should keep the previous way of documenting `custom_queries`, it seems more convenient for user by providing a clear example.